### DISCUSSION TITLE
Hardware you tested this on

### DISCUSSION THREAD

**Discussion by OldDudeDallas** [ekondayan/libiolink#2] [Q&A]
What hardware have you tested this on. I am thinking of using something with just ethernet port so I can talk to a Keyence laser, so this is pretty interesting to me.

**Comment by ekondayan** [comment]
I've tested it with AL1352 master. I do not have access to other ifm IO-Link masters but they should be easily adapted. I've checked other masters datasheets and they seem quite identical.

As for the devices, I've tested only those in the src/driver/device/ sub-folder. If the device has good datasheet and documentation, it is quite straightforward to implement a driver for it.

You can also consider the MQTT protocol. I personally find it stable enough for production.

**Comment by OldDudeDallas** [comment]
Thank you very much!

 

Thanks,

 

Rod Longhofer

 

Rodney Longhofer 

Micropac Industries, Inc.

Address: 905 E Walnut St., Garland, TX 75040

Email:  ***@***.***> ***@***.***

Office: 972-272-3571, Ext. 1259

Cell: 972-730-1963

 



 

      <http://www.micropac.com/> www.micropac.com

 

 

 

This message, including any attachments, may contain information that is confidential and proprietary information of Micropac Industries, Inc.  The dissemination, distribution, use or copying of this message or any of its attachments is strictly prohibited without the express written consent of Micropac Industries, Inc. If you are not the intended recipient of this message, you are kindly requested to inform the sender and then to please delete the email and any attachments.
